RecoMuon/MuonSeedGenerator/src/MuonSeedPtExtractor.cc

```diff
@@ -391,6 +391,16 @@ int MuonSeedPtExtractor::stationCode(MuonTransientTrackingRecHit::ConstMuonRecHi
     if(result == 1 && (1 == cscID.ring() ||  4 == cscID.ring()) )
        result = 0;
   }
+  else if( hit->isGEM() ){
+    GEMDetId gemID(detId);
+    //std::cout<<"first (GEM) E/S/R/C = "<<gemID.endcap()<<"/"<<gemID.station()<<"/"<<gemID.ring()<<"/"<<gemID.chamber()<<std::endl;
+    result = gemID.station();
```

Comment:
shouldn't this be changed to ```std::min(2, gemID.station())``` to match the logic in 
https://github.com/cms-sw/cmssw/pull/18625/files#diff-31670b17a03a26fbe9f97392ad91a94cR41
?

Similarly, for ME0 in  RecoMuon/MuonSeedGenerator/src/MuonCSCSeedFromRecHits.cc  there is a hardcoded station = 1.
Is it OK to use explicit station value here and not there?